### PR TITLE
Add messaging for setup without a custom domain

### PR DESCRIPTION
**Pull request by secade**
In response to #1316 

* Adds instructions for lacking a custom DNS name *before* inputting anything for AWS

Extra:

* Added a symlink for a top-level `CONTRIBUTING.MD`as the contributing instructions are nested under the docs folder

### CODE REVIEW

File: pkg/jx/cmd/init.go [hunk 1]
```diff
@@ -709,6 +709,8 @@ func (o *CommonOptions) GetDomain(client kubernetes.Interface, domain string, pr
 			return defaultDomain, nil
 		}
 		log.Successf("You can now configure a wildcard DNS pointing to the new loadbalancer address %s", address)
+		log.Info("\nIf you do not have a custom domain setup yet, Ingress rules will be set for magic dns nip.io.")
+		log.Infof("\nOnce you have a customer domain ready, you can update with the command %s", util.ColorInfo(ingressNamespace))
```
Comment (rawlingsj) [inline pkg/jx/cmd/init.go]: I think the value substitution here should be `jx upgrade ingress --cluster`

Comment (secade) [reply pkg/jx/cmd/init.go]: oops, bad copypasta. will update.

**Comment by rawlingsj** [comment]
Great thanks @secade!  Just the one comment as I wasn't 100% sure on the fmt message part.

**Comment by secade** [comment]
@rawlingsj Thanks for the catch, just had some bad copypasta and forgot to include the actual instruction.

**Comment by rawlingsj** [comment]
Np - thanks for the PR! :)

**Pull request merged**